### DISCUSSION TITLE
公式的处理

### DISCUSSION THREAD

**Discussion by PirateBook1** [banned-historical-archives/banned-historical-archives.github.io#1282] [Q&A]
![image](https://user-images.githubusercontent.com/113572850/203703324-6e258cc6-f28a-472b-9a8d-f7140fe8e075.png)
@banned-historical-archives 《自然辩证法》一书中出现了公式，又是一个等待处理的问题

**Comment by banned-historical-archives** [comment]
简单的公式建议这样处理 v=√(2gh)，(mv^2)/2。

**Reply by PirateBook1** [reply]
不能采用这种治标不治本的态度，如果有微分和积分符号又该怎么办呢

**Reply by banned-historical-archives** [reply]
再复杂要用latex，可能需要新增一种段落类型math。遇到再考虑吧

**Reply by PirateBook1** [reply]
《资本论》里也大量出现了公式

**Reply by banned-historical-archives** [reply]
资本论里的公式应该不会太复杂

**Reply by PirateBook1** [reply]
还是把 LateX 的开发安排上吧，一劳永逸解决公式问题

**Reply by PirateBook1** [reply]
《反杜林论》里出现了一些微分学符号

**Comment by PirateBook1** [comment]
![image](https://user-images.githubusercontent.com/113572850/212632877-8d5e236b-4eff-49f5-9ba4-77fe326a69b8.png)
@banned-historical-archives

**Comment by CiangCing14** [comment]
这些公式之类的，最好学会使用Tex Studio，有些特殊的LaTeX文字，估计公式识别的AI也没有办法，比如《反对党八股》里的人三这个字。

**Reply by CiangCing14** [reply]
Tex Works。

**Reply by CiangCing14** [reply]
因为asciimath似乎不支持各种化学式。

**Reply by CiangCing14** [reply]
我说的是一个专业的LaTeX工具，专门用来制作文档PDF的，也用来制作特殊格式。
